store/src/java/com/zimbra/cs/util/Zimbra.java

```diff
@@ -404,6 +409,7 @@ public static synchronized void shutdown() throws ServiceException {
 
         if (sIsMailboxd) {
             PurgeThread.shutdown();
+            ContactBackupThread.shutdown();
```

Comment:
Do we need to check ContactBackupThread support here?

Reply:
yes